client-code-gen/package.json

```diff
@@ -3,6 +3,7 @@
   "version": "1.0.0",
   "description": "OpenAPI Generation for API Client Code",
   "scripts": {
+    "gen-api-docker": "docker run --rm -v ${PWD}:/local openapitools/openapi-generator-cli:v6.2.1 generate -i local/openapi.json -g typescript-axios --config local/api.json -o /local/gen",
```

Comment:
This might not work on Windows, I'd probably just map the current directory as this would normally be run from the client-code-gen directory anyway. Or is this not true for CI build?

Reply:
Good point, sorry, I am using Windows WSL system and did not think about cross-platform.
I don't think it would work for Windows. I will need to do new checkout for Windows and try.
However, this might be a hard question (as it always is for cross-platform) as for now I feel using "./" to map current directory would work, may need to find out if there is a simple trick.
Will do some experiment and let you know. Worst case I think, is create another npm script for Windows (on Powershell) to work for code generation on Windows.

Reply:
You could have a second npm script that just uses the local directory, and keep gen-api-docker for the CI build (maybe rename accordingly).

Reply:
Yes, that's what I thought easiest option and I added the Windows option in (tested and worked for both Powershell and CMD prompt). Please have a look.